torch/csrc/distributed/rpc/python_functions.h

```diff
@@ -41,6 +41,10 @@ PyRRef pyRemotePythonUdf(
     std::vector<torch::Tensor>& tensors,
     const std::shared_ptr<torch::autograd::profiler::RecordFunction>& rf);
 
+void remoteCallCallback(
```

Comment:
I am not sure if this function fit in here. All other functions are exposed to Python as RPC entry points to C++, but not this one. Maybe expose it on RRefContext? Or maybe we can tolerate a little code duplication here?

Reply:
Moved to rref_context.h/cpp, and put it into a "callback" namespace.